package.json

```diff
@@ -33,7 +33,6 @@
     "docker:puppeteer:windows": "docker-compose -f ./support/docker/docker-compose.yml build puppeteer-windows",
     "docker:test:puppeteer": "docker-compose -f ./support/docker/docker-compose.yml run puppeteer-linux yarn test:puppeteer",
     "docker:test:puppeteer:firefox": "docker-compose -f ./support/docker/docker-compose.yml run puppeteer-linux yarn test:puppeteer:firefox",
-    "docs": "cd docs && yarn dev --debug",
```

Comment:
[contributing.md](https://github.com/ifiokjr/remirror/blob/master/docs/contributing.md) will need to be updated. IS the new intended command `yarn build:docs` ?

Reply:
Good point - it probably shouldn't be removed.